app-emulation/containerd/files/containerd.service

```diff
@@ -4,7 +4,7 @@ Documentation=http://github.com/docker/containerd
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc
+ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc $CONTAINERD_OPTS
```

Comment:
Why not break all of these flags out?

What do you intend to add to CONTAINERD_OPTS?